demo/config.yaml

```diff
@@ -20,3 +20,31 @@ lcr-modules:
         inputs:
             sample_bam: "data/{sample_id}.bam"
             sample_bai: "data/{sample_id}.bam.bai"
+
+    liftover:
+        inputs:
+            sample_seg: "data/{sample_id}_subclones.hg38.igv.seg"
+        scratch_subdirectories: "scratch/"
+        options:
+            seghg38tobedhg38: "{MODSDIR}/src/convert_for_liftover.py"
+            bedhg38tobedhg19: "{MODSDIR}/src/chains/hg38ToHg19.over.chain"
+            bedhg19toseghg19: "{MODSDIR}/src/convert_for_liftover.py"
+        conda_envs:
+            liftover-366: "{MODSDIR}/envs/liftover-366.yaml"
+        threads:
+            seghg38tobedhg38: 12
+            bedhg38tobedhg19: 12
+            bedhg19toseghg19: 12
+        mem_mb:
+            seghg38tobedhg38: 100000
```

Comment:
It does not require that much memory. Even if I limit to only 1 core, it finishes running the test file in less than 5 sec, so even if I do `| htop` , I still can't see how much memory exactly is used. I think that once it will be tested on a large number of files,  it will be better to estimate memory usage and then specify `mem_mb` value.